components/Counters/Counters.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import Counter from "./Counter";
+import { CountersContainer } from "./styles";
+
+function Counters() {
+  const [boxes, setBoxes] = useState([0, 0]);
+  const [buckets, setBuckets] = useState([0, 0]);
+
+  return (
+    <div>
+      <CountersContainer>
+        {boxes.map((value, index) => (
+          <Counter
+            key={index}
+            name={`Kisten ${index === 0 ? "geliefert:" : "zurück:"}`}
+            value={value}
+            onIncrement={() => {
+              const boxesCopy = [...boxes];
+              boxesCopy[index] += 1;
+              setBoxes(boxesCopy);
+            }}
+            onDecrement={() => {
+              const boxesCopy = [...boxes];
+              boxesCopy[index] -= 1;
+              setBoxes(boxesCopy);
+            }}
+          />
+        ))}
+      </CountersContainer>
+      <CountersContainer>
+        {buckets.map((value, index) => (
+          <Counter
+            key={index}
+            name={`Eimer ${index === 0 ? "geliefert:" : "zurück:"}`}
+            value={value}
+            onIncrement={() => {
+              const bucketsCopy = [...buckets];
```

Comment:
shouldn't 
```suggestion
              const bucketsCopy = buckets;
```
also work?